sensor_msgs/PointCloud2 point_cloud
---
sensor_msgs/PointCloud2[] detected_objects

